wisembly/amqp-bundle/1.3/config/packages/amqp.yaml

```diff
@@ -0,0 +1,16 @@
+wisembly_amqp:
+    console_path: '%kernel.project_dir%/bin/console'
+    broker: 'oldsound'
+    connections:
+        default: '%env(AMQP_URI)%'
```

Comment:
We are using `DATABASE_URL` or `MAILER_URL`, so what about using `AMQP_URL` here?

Reply:
Changed.